src/api_manager/service_control/aggregated.cc

```diff
@@ -377,21 +377,10 @@ void Aggregated::Check(
     if (status.ok()) {
       Status status = Proto::ConvertCheckResponse(
           *response, service_control_proto_.service_name(), &response_info);
-      // If server replied with either invalid api_key or not activated service,
-      // the request is rejected even allow_unregistered_calls is true. Most
-      // likely, users provide a wrong api key. By failing the request, the
-      // users will be notified with the error and have chance to correct it.
-      // Otherwise, the Report call will fail. It is very hard to notice and
-      // debug the Report failure.
-      if (allow_unregistered_calls && response_info.is_api_key_valid &&
```

Comment:
This change is just clean up the code, did not change any logic.  "allow_unregistered_calls" is always false.   If it is true,  The Check is NOT called at tall.